monkey/infection_monkey/post_breach/job_scheduling/windows/job_scheduling.py

```diff
@@ -0,0 +1,28 @@
+SCHEDULED_TASK_NAME = 'monkey-spawn-cmd'
+SCHEDULED_TASK_COMMAND = 'C:\windows\system32\cmd.exe'
+
+
+def get_windows_commands_to_schedule_jobs():
+    return [
+        'schtasks',
+        '/Create',
+        '/SC',
+        'monthly',
+        '/TN',
+        SCHEDULED_TASK_NAME,
+        '/TR',
+        SCHEDULED_TASK_COMMAND
```

Comment:
Not very readable. From human perspective this is a command, not an array. I think subprocess can take a string as a command. If not, then maybe lets return `" ".split(schtasks /Create /SC`